CMakeLists.txt

```
@@ -17,13 +17,20 @@
 
 cmake_minimum_required(VERSION 3.5)
 
+# build the sdk targets
+project(olp-cpp-sdk VERSION 0.6.0)
+
 # Options
 option(EDGE_SDK_ENABLE_TESTING "Flag to enable/disable building unit and integration tests" ON)
 option(EDGE_SDK_BUILD_DOC "Build SDK documentation" OFF)
 option(EDGE_SDK_NO_EXCEPTION "Turn off EXCEPTION" OFF)
 option(EDGE_SDK_BUILD_EXTERNAL_DEPS "Download and build external dependencies" ON)
 option(EDGE_SDK_BUILD_EXAMPLES "Enable examples targets" OFF)
 
+if (MSVC)
+option(EDGE_SDK_PARALLEL_BUILD_ENABLE "Enable parallel build on MSVC" ON)
```

Better name it platform dependent, like EDGE_SDK_MSVC_PARALLEL_BUILD_ENABLE:
```suggestion
option(EDGE_SDK_MSVC_PARALLEL_BUILD_ENABLE "Enable parallel build on MSVC" ON)
```

Btw, do we really need this flag? Can't the user just pass /MP flag to CMAKE_CXX_FLAGS?

---

Done